MovieApp/MovieApp/FeedDownloader.h

```diff
@@ -0,0 +1,25 @@
+//
+//  FeedDownloader.h
+//  MovieApp
+//
+//  Created by user on 26/09/16.
+//  Copyright © 2016 internshipABH. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+#import "NewFeedsItem.h"
+
+@protocol FeedDataReceiver <NSObject>
+
+@required
+-(void)updateViewWithNewData:(NSMutableArray *)feedItemsArray;
+
+@end
+
+@interface FeedDownloader : NSObject <NSURLConnectionDelegate, NSXMLParserDelegate>
+
+@property (strong, nonatomic) NSURL * feedURL;
```

Comment:
Property `feedURL` should not be accessed directly, since it's used only by `FeedDownloader` class. Keep in mind what properties you will expose and if they should be readonly.